components/NewNFTBook.vue

```diff
@@ -956,8 +982,43 @@ async function submitEditedClass () {
   }
 }
 
-function onSubmit () {
-  return isEditMode.value ? submitEditedClass() : submitNewClass()
+async function addNewEdition () {
+  try {
+    isLoading.value = true
+    const p = mapPrices(prices.value)
+    const price = p[0]
+    const autoDeliverCount = p
+      .filter(price => price.isAutoDeliver)
+      .reduce((acc, price) => acc + price.stock, 0)
```

Comment:
`price` is not use until 1010, move `const price = p[0]` to line 1010 to avoid variable shadowing.

Reply:
[🎨 Improve condition checks and price declaration](https://github.com/likecoin/nft-book-press/pull/332/commits/91bc387a398467f0a79548cf4bfd12dd77ce4c5f)